Q1.py

```diff
@@ -0,0 +1,16 @@
+# [“name512”, “same1example”, “joy18full”] replace the digits from string inside list
+
+lst = ['name512', 'same1example', 'joy18full']
+count = 0   
+for i in lst:
```

Comment:
you can use `enumerate` instead to access both index and value from iterables. would save you from maintaining `count` variable.

Reply:
I have pushed code again where I am utilizing enumerate. Please review.